clients/venice-samza/src/main/java/com/linkedin/venice/samza/VeniceSystemProducer.java

```diff
@@ -185,25 +185,25 @@ public String getRunningFabric() {
     return this.runningFabric;
   }
 
-  protected ControllerResponse controllerRequestWithRetry(Supplier<ControllerResponse> supplier) {
+  protected ControllerResponse controllerRequestWithRetry(Supplier<ControllerResponse> supplier, int retryLimit) {
```

Comment:
Can we make this method better with [exponential backoff and jitter algorithm](https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/)?
Maybe no jitter for now, but I think exponential backoff would be beneficial to controller.

Reply:
It already has exponential backoff. 
  time.sleep(1000L * (currentAttempt + 1));

Reply:
Ack. I missed it in this method. Was hoping it can be configurable in the method signature. Resolving this comment.